continuous_integration/scripts/check_mccabe_complexity.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I'm a bit surprised by this manual script, isn't there some higher level tooling to prevent to create such a script in each repo?